packages/layout/src/components/beta/beta.tsx

```diff
@@ -14,11 +14,13 @@ type MailToProps = {
 
 export const BetaHeader: React.FC<BetaHeaderProps> = ({ text, mail }) => {
 	const TextComponent = useMemo(() => {
-		return text
-			? () => <>{text}</>
-			: () => (
+		return () => (
+			<Span cfg={{ fontSize: 2, color: 'neutral.8', my: 1 }}>
+				{text ? (
+					text
+				) : (
 					<>
-						This is a new service - your{' '}
+						This is a new service &mdash; your{' '}
```

Comment:
It's a minor thing but now that we're in the age of Unicode you don't need to use an entity reference for special characters, you can just paste it in and it'll save a few bytes. Here's one: —